### DISCUSSION TITLE
Creators with follow disabled?

### DISCUSSION THREAD

**Discussion by Chrdvr** [Avnsx/fansly#153] [General]
I have a creator I am subscribed to who has a decent amount on their page, but has follow disabled. The tool is only picking up about 15 images and 4 videos from one recent index, then can't find any more. Curious if this is normal when follow is disabled even though I can see all the content as a subscriber? Thanks!

**Comment by Avnsx** [comment]
https://github.com/Avnsx/fansly-downloader/issues/148

**Reply by Chrdvr** [reply]
Perfect I'll watch for the update!
